File: problems/1748-sum-of-unique-elements/solution.java
class Solution {
    public int sumOfUnique(int[] nums) {
        int[] count = new int[101];
        int sum = 0;
        for (int num : nums) {
            count[num]++;
        }
        for (int i = 1; i <= 100; i++) {
            if (count[i] == 1) {
                sum += i;
            }
        }
        return sum;
    }
}

File: problems/1748-sum-of-unique-elements/solution.py
class Solution:
    def sumOfUnique(self, nums: List[int]) -> int:
        counts = {}
        for num in nums:
            counts[num] = counts.get(num, 0) + 1
        
        unique_sum = 0
        for num, count in counts.items():
            if count == 1:
                unique_sum += num
        
        return unique_sum